packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -331,6 +331,30 @@ function isContainerConnectionStatusInProgress(
               </div>
             </div>
           {/each}
+          {#each provider.kubernetesConnections as kubeConnection}
+            <div class="px-5 py-2 w-[240px]">
+              <div class="text-sm">
+                {kubeConnection.name}
+              </div>
+              <div class="flex mt-1">
+                {#if kubeConnection.status === 'started'}
+                  <div class="my-auto w-3 h-3 bg-green-500 rounded-full"></div>
+                  <span class="my-auto text-green-500 ml-1 font-bold text-[9px]">RUNNING</span>
+                {:else}
+                  <div class="my-auto w-3 h-3 bg-gray-500 rounded-full"></div>
+                  <span class="my-auto text-gray-500 ml-1 font-bold text-[9px]">OFF</span>
```

Comment:
in the API we have status for
```
'started' | 'stopped' | 'starting' | 'stopping' | 'unknown'
```

and we only report `RUNNING` or `OFF`
I guess would be interesting to have starting/stopping

in containers we do report STARTING but not stopping

so at the end it looks we are half copy/pasting the same stuff so probably
https://github.com/containers/podman-desktop/blob/86c0a6dc168386bf626c0edcd42acc863186faa2/packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte#L233-L246

should be a widget shared by both Kube and Container objects ?